static/dtale/menu/XArrayOption.jsx

```diff
@@ -21,35 +22,32 @@ function renderDimensionSelection(dimensionSelection) {
 class ReactXArrayOption extends React.Component {
   constructor(props) {
     super(props);
+    this.buttonRef = React.createRef();
```

Comment:
I don't think you need this `buttonRef` isn't that all handled within `MenuItem`?

Reply:
100% right, hat's a mistake that I forgot to remove :) will get that